### DISCUSSION TITLE
[General] Bridging error.

### DISCUSSION THREAD

**Discussion by galacticminter** [zkSync-Community-Hub/zksync-developers#719] [General] [answered]
### Environment

Mainnet

### L2 block number

 45598363

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hi, I am withdrawing ETH from ZkSync to Eth mainnet. However, I am running the issue on the UI where it doesn't let me complete the transaction on the mainnet side. 

### Additional Details

Withdraw txs on zkSync - https://era.zksync.network/tx/0x11b4b00cb03c6959e98cdb2f55ce9795a9cb324e4566eb4518b12b580aa8613f

Picture of the error.
```
entry.MrNZu0eh.js:13  ContractFunctionExecutionError: The contract function "finalizeWithdrawal" reverted with the following reason:
xx

Contract Call:
  address:   0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb
  function:  finalizeWithdrawal(uint256 _chainId, uint256 _l2BatchNumber, uint256 _l2MessageIndex, uint16 _l2TxNumberInBatch, bytes _message, bytes32[] _merkleProof)
  args:                        (324, 492711, 1, 1002, 0x6c0960f93924a9a1706285f5e92486dc19945e43fb2f98cf000000000000000000000000000000000000000000000000b51ae9dbaed90000, ["0x4929459ecbf076b3eddb8a27d4c97eaae4da85561be79c54867c5b14f5411c44","0xadb2754135748271ad2aadde974a773f995be0182cdedbd6f64c4f73d1ad0a31","0xe3697c7f33c31a9b0f0aeb8542287d0d21e8c4cf82163d0c44c7a98aa11aa111","0x199cc5812543ddceeddd0fc82807646a4899444240db2c0d2f20c3cceb5f51fa","0xe4733f281f18ba3ea8775dd62d2fcd84011c8c938f16ea5790fd29a03bf8db89","0x1798a1fd9c8fbb818c98cff190daa7cc10b6e5ac9716b4a2649f7c2ebcef2272","0x66d7c5983afe44cf15ea8cf565b34c6c31ff0cb4dd744524f7842b942d08770d","0xb04e5ee349086985f74b73971ce9dfe76bbed95c84906c5dffd96504e1e5396c","0xac506ecb5465659b3a927143f6d724f91d8d9c4bdb2463aee111d9aa869874db","0x124b05ec272cecd7538fdafe53b6628d31188ffb6f345139aac3c3c1fd2e470f","0xc3be9cbd19304d84cca3d045e06b8db3acd68c304fc9cd4cbffe6d18036cb13f","0xfef7bd9f889811e59e4076a0174087135f080177302763019adaf531257e3a87","0xa707d1c62d8be699d34cb74804fdd7b4c568b6c1a821066f126c680d4b83e00b","0xf6e093070e0389d2e529d60fadb855fdded54976ec50ac709e3a36ceaa64c291"])
  ```

**Comment by alhenaai** [comment]
It appears you are encountering a "ContractFunctionExecutionError" during the withdrawal process from ZKsync to the Ethereum mainnet. This error typically indicates an issue with the smart contract function `finalizeWithdrawal` on the Ethereum side.

Here are a few steps you can take to troubleshoot and potentially resolve this issue:

1. **Check Network Congestion:** Sometimes, network congestion on Ethereum can cause transactions to fail or not be processed as expected. Verify the current network status to see if there might be delays.

2. **Gas Fees and Parameters:** Ensure that the gas fees and other transaction parameters are correctly set according to the current network conditions. Insufficient gas can cause transactions to revert.

3. **Contract Address and Function:** Double-check the contract address and the parameters being passed to the `finalizeWithdrawal` function. There might be an issue with the inputs provided.

4. **Retry the Transaction:** If the network conditions are stable and the parameters are correct, try resubmitting the transaction. Sometimes, resubmission can resolve temporary issues.

5. **Support and Further Assistance:** If the problem persists, consider reaching out for more personalized support. You can contact the ZKsync support team or consult the community on platforms like Discord for more detailed guidance.

If you need further assistance or have additional questions, feel free to reach out!

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by galacticminter** [comment]
I got this withdrawal cleared by interacting with the contract directly but couldn't get it to work based on UI despite multiple browsers or wallets.

**Reply by bxpana** [reply, marked as answer]
Glad you were able to get it working, but right now with the UI it's a known issue https://github.com/zkSync-Community-Hub/zksync-developers/discussions/711#discussioncomment-10812935
